src/containers/setContacts/actions.ts

```diff
@@ -0,0 +1,8 @@
+import { genericAsyncActions } from '../../utils/asyncRequest';
+
+export const setContacts = genericAsyncActions<void, void>();
```

Comment:
as we discussed, this won't be void, the payload will be of the type of the response.